dev-requirements.txt

```diff
@@ -0,0 +1,58 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file=dev-requirements.txt dev-requirements.in
```

Comment:
I guess the correct command to execute the compilation is `pip-compile --pre --output-file=dev-requirements.txt dev-requirements.in` -

The generated files will look the same in your setup since you're using only tensorflow 2.0 (that is still a pre-release package), but if someone else runs this command it will gather the latest stable TensorFlow package (1.14)

Reply:
Ok, done